### PR TITLE
Chore(arkmarket): add starkscan link to token activity and collection activity tables issue #167

### PR DESCRIPTION
**Pull request by stephanniegb**

### CODE REVIEW

File: apps/arkmarket/src/app/collection/[collectionAddress]/components/desktop-collection-activity.tsx [hunk 1]
```diff
@@ -206,7 +206,7 @@ export default function DesktopCollectionActivity({
               </TableCell>
               <TableCell className="pr-5">
                 <Button asChild size="icon" variant="outline">
-                  <ExternalLink href="/">
+                  <ExternalLink href={`https://starkscan.co/nft/${activity.address}/${activity.token_id}`}>
```
Comment (YohanTz) [inline apps/arkmarket/src/app/collection/[collectionAddress]/components/desktop-collection-activity.tsx:209]: Hey @stephanniegb, thanks for your contribution

This link should also point to the transaction itself, as you did in `desktop-portfolio-activity`

**Comment by stephanniegb** [comment]
Should I disable the buttons of those activities that don't have a transaction hash?

**Comment by YohanTz** [comment]
> Should I disable the buttons of those activities that don't have a transaction hash?

Mmh good question, you can just show an empty cell in this case @stephanniegb

File: apps/arkmarket/src/app/collection/[collectionAddress]/components/desktop-collection-activity.tsx [hunk 2]
```diff
@@ -205,11 +205,17 @@ export default function DesktopCollectionActivity({
...
+                {activity.transaction_hash ? (
+                  <Button size="icon" variant="outline" asChild>
+                    <ExternalLink
+                      href={`https://starkscan.co/tx/${activity.transaction_hash}`}
+                    >
+                      <ArrowUpRight className="size-5" />
+                    </ExternalLink>
+                  </Button>
+                ) : (
+                  "_"
```
Comment (YohanTz) [inline apps/arkmarket/src/app/collection/[collectionAddress]/components/desktop-collection-activity.tsx:217]:
```suggestion
                  null
```

**Comment by YohanTz** [comment]
> > Should I disable the buttons of those activities that don't have a transaction hash?
> 
> Mmh good question, you can just show an empty cell in this case @stephanniegb

@stephanniegb Could you do this for both cells? 🙏

**Review by YohanTz — APPROVED** [review]
LGTM!

**Pull request merged**